R/cosh_nlsfit.R

```diff
@@ -161,7 +161,8 @@ fit.cosh <- function(effMass, cf, t1, t2, useCov=FALSE, m.init, par, n.cosh=2, a
   return(invisible(effMass))
 }
 
-plot.coshfit <- function(effMass, col.fitline="black", plot.mass=TRUE, plot.corr=FALSE, ...) {
+plot.coshfit <- function(x, col.fitline = "black", plot.mass = TRUE, plot.corr = FALSE, ...) {
+  effMass <- x
```

Comment:
***must*** the first argument really be named ```x```?

Reply:
Yes, I think that it is because one wants to support named arguments. Say a user writes the generic `plot(x = samplecf)`, then this will fail with the current `plot.cf` because it expects a first argument called `cf`:

```
> plot(x = samplecf)
Fehler in any(inherits(cf, c("cf_orig", "cf_boot", "cf_jackknife"))) : 
  Argument "cf" fehlt (ohne Standardwert)
```

Having different argument names might be nice in the context of the particular function, but it breaks named arguments in the large picture.

Reply:
I see, it does make it ugly in the function, especially because 'x' is used elsewhere for the horizontal coordinate...

Reply:
With the `effMass <- x` we give that a proper name and free up `x` for use in the function, so perhaps the current state is not that bad after all for this function?

Reply:
I support this renaming...!